client/source/components/questionsAnswers/HelpfulQ.jsx

```diff
@@ -12,6 +22,29 @@ const HelpfulQ = function CreateHelpfulQComponent({ questionHelpfulness, id, get
         getQuestions();
       })
       .catch((err) => console.error('handleHelpfulQ error: ', err));
+  }, [id]);
+
+  const postAnswer = (data) => {
+    const obj = {
+      body: data.answer, name: data.name, email: data.email, photos: data.photos
+    };
+    return requests
+      .post(`/qa/questions/${id}/answers`, obj);
+  };
+
+  const handlePost = (event, input) => {
+    const keys = Object.keys(input);
+    for (let i = 0; i < keys.length; i += 1) {
+      if (input[keys[i]] === '') {
+        alert(`You must enter the following: ${keys[i]}`);
+      }
+    }
+    postAnswer(input)
```

Comment:
I think when someone clicks out of the alert, it continues to post without the user actually updating the field that needs to have input.

Reply:
Fixed this too, so it should stop warning now after first non-updated field